must-call-qual/src/main/java/org/checkerframework/checker/mustcall/qual/MustCallTop.java

```diff
@@ -0,0 +1,16 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * The top qualifier in the Must Call type hierarchy. It represents a type that must have all of its
```

Comment:
Most of this comment is irrelevant because of the incorrect documentation of the type hierarchy. However, I agree that `*Top` isn't a very useful name. In this type system's case, though, `MustCallUnknown` wouldn't be right - it's more like "must-call unknowable/unsatisfiable". I therefore decided to rename it `MustCallAll` and add documentation that indicated that it can never be satisfied (because the "all" it represents really means "all possible methods", not just defined on the class).